jetty-osgi/test-jetty-osgi/src/test/java/org/eclipse/jetty/osgi/test/TestJettyOSGiBootHTTP2Conscrypt.java

```diff
@@ -99,7 +101,10 @@
         List<Option> res = new ArrayList<>();
         res.add(CoreOptions.systemProperty("jetty.alpn.protocols").value("h2,http/1.1"));
         res.add(CoreOptions.systemProperty("jetty.sslContext.provider").value("Conscrypt"));
-        
+
+        String conscryptVersion = System.getProperty("conscrypt-version");
+        Assume.assumeFalse("Missing 'conscrypt-version' system property", (conscryptVersion == null || conscryptVersion.equals("null")));
+
```

Comment:
Nothing to do with the servlet-api change?

Reply:
reverted.